### PR TITLE
{C++}Clarified fooRef address.

### PR DESCRIPTION
**Pull request by thePsguy**
Regarding issue #1216, Better explaining the Reference 'fooRef'.

### CODE REVIEW

File: c++.html.markdown [hunk 1]
```diff
@@ -245,7 +245,13 @@ cout << fooRef; // Prints "I am foo. Hi!"
 // Doesn't reassign "fooRef". This is the same as "foo = bar", and
 //   foo == "I am bar"
 // after this line.
+cout << &fooRef << endl; //Prints address of fooRef
 fooRef = bar;
+cout << &fooRef << endl; //Prints address of fooRef, AGAIN
```
Comment (vendethiel) [inline c++.html.markdown]: shouldn't that be "still"?

Comment (thePsguy) [inline c++.html.markdown]: My bad. Also it prints address of foo, fooRef is just a reference. Fixed it.

Comment (thePsguy) [inline c++.html.markdown]: My bad. Also it prints address of foo, fooRef is just a reference. Fixed it.

File: c++.html.markdown [hunk 2]
```diff
@@ -245,7 +245,13 @@ cout << fooRef; // Prints "I am foo. Hi!"
 // Doesn't reassign "fooRef". This is the same as "foo = bar", and
 //   foo == "I am bar"
 // after this line.
+cout << &fooRef << endl; //Prints address of foo
 fooRef = bar;
+cout << &fooRef << endl; //Still prints address of foo
```
Comment (vendethiel) [inline c++.html.markdown]: **the** address

Comment (thePsguy) [inline c++.html.markdown]: oops! Edited.

**Comment by vendethiel** [comment]
Yay! Nice!

**Comment by vendethiel** [comment]
:yellow_heart:

**Comment by thePsguy** [comment]
Awesome! :D

**Pull request merged**